lib/Installation/Registration/RegistrationController.pm

```diff
@@ -28,11 +28,9 @@ sub init {
     my ($self, $args) = @_;
     $self->{RegistrationPage}    = Installation::Registration::RegistrationPage->new({app => YuiRestClient::get_app()});
     $self->{UseUpdateReposPopup} = Installation::Warnings::ConfirmationWarning->new({
-            app        => YuiRestClient::get_app(),
-            match_text => "The registration server offers update repos.*"});
```

Comment:
Why is `match_text` also deleted?

Reply:
Ah I see, probably was used in `has_valid_text` and now has no usage as well

Reply:
Because it was just used by `has_valid_text` method. If no method, then no 'match_text' needed:)